react/components/MicroData.js

```diff
@@ -49,17 +52,22 @@ const tryParsingLocale = (description, locale) => {
   return parsedDescription || description
 }
 
-export default function MicroData({ product }, { culture: { currency, locale } }) {
-  const { image, seller } = lowestPriceInStock(product)
+
+export default function MicroData({ product, query }, { culture: { currency, locale } }) {
+  const skuId = query.skuId ? query.skuId : product.items[0].itemId
```

Comment:
I think `const skuId = query.skuId || product.items[0].itemId` would do the trick

Reply:
done